src/lbfgs.jl

```diff
@@ -27,6 +27,22 @@ The keyword arguments may include
 # Output
 The returned value is a `GenericExecutionStats`, see `SolverCore.jl`.
 
+# Callback
+The callback is called after each iteration.
```

Comment:
```suggestion
The callback is called at each iteration.
```

Reply:
Thanks for the feedback. I also applied this to R2.